code401Challenges/src/main/java/code401Challenges/hashtable/Hashtable.java

```diff
@@ -0,0 +1,79 @@
+package code401Challenges.hashtable;
+
+
+
+public class Hashtable<V> {
+
+    // Instance variables
+    // Set to generic for different possible key:value pairs
+
+    private Node<String, V> [] map;
+    private final int initialSize = 1024;
+    private final int primeValue = 4096;
+
+    //Constructor
+
+    public Hashtable() {
+
+        this.map = new Node[initialSize];
+    }
+
+    // Methods
+
+    //Hash the key
+    public int hash(String key){
+        int hashValue = 0;
+
+        for (int i = 0; i < key.length(); i++){
+            hashValue += ( (int) key.charAt(i) + primeValue) * 3;
+        }
+
+        return hashValue % map.length;
+    }
+
+    // Add the key value to the hash table
+    public void add(String key, V value){
+        Node <String, V> node = new Node<>(key, value, null);
+        int hashKey = hash(key);
+        if (this.map[hashKey] != null) {
+
+            Node temp = map[hashKey];
+            map[hashKey] = node;
+            map[hashKey].setNext(temp);
```

Comment:
This throws a warning because you have a specific type of node put into an array of unspecified nodes. Always specific, even with generics!